src/main/java/io/mosip/mimoto/exception/PlatformErrorMessages.java

```diff
@@ -86,7 +86,11 @@ public enum PlatformErrorMessages {
     MIMOTO_ISSUER_ONBOARDING_EXCEPTION(PlatformConstants.PREFIX + "037", "Issuers Onboarding exception occurred "),
     INVALID_VERIFIER_ID_EXCEPTION(PlatformConstants.PREFIX + "038", "Invalid Verifier ClientId"),
     INVALID_VERIFIER_REDIRECT_URI_EXCEPTION(PlatformConstants.PREFIX + "039", "Invalid Redirect Uri"),
-    INVALID_CREDENTIAL_RESOURCE_URI_EXCEPTION(PlatformConstants.PREFIX + "040", "The Credential Resource is Invalid");
+    INVALID_CREDENTIAL_RESOURCE_URI_EXCEPTION(PlatformConstants.PREFIX + "040", "The Credential Resource is Invalid"),
+    PROOF_DOCUMENT_NOT_FOUND_EXCEPTION(PlatformConstants.PREFIX + "041", "Proof document is not available in the received credentials" ),
+    PROOF_TYPE_NOT_FOUND_EXCEPTION(PlatformConstants.PREFIX + "042", "Proof Type available in received credentials is not matching with supported proof terms" ),
+    SIGNATURE_VERIFICATION_EXCEPTION(PlatformConstants.PREFIX + "043", "Error while doing signature verification" ),
+    UNKNOWN_EXCEPTION(PlatformConstants.PREFIX + "044", "Error while doing verification of verifiable credential" );
```

Comment:
Instead of error code like this, can we start using ErrorConstants https://github.com/mosip/mimoto/blob/develop/src/main/java/io/mosip/mimoto/exception/ErrorConstants.java

Reply:
not changing as discussed